File: testprojects/scala_main/pipelines/PERF_SC_PIP/code/src/main/scala/org/main/scla_dep_mgmt_change/graph/PERF_SQLSTATEMENT.scala
package org.main.scla_dep_mgmt_change.graph

import io.prophecy.libs._
import org.main.scla_dep_mgmt_change.config.ConfigStore._
import org.main.scla_dep_mgmt_change.config.Context
import org.main.scla_dep_mgmt_change.udfs.UDFs._
import org.main.scla_dep_mgmt_change.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object PERF_SQLSTATEMENT {

  def apply(context: Context, in0: DataFrame): DataFrame = {
    in0.createOrReplaceTempView("in0")
    context.spark.sql("select * from in0 where first_name not like '%123%'")
  }

}

File: testprojects/python_main/pipelines/PERF_PY_PIP/code/com/main1/pythondepmanagement_1/graph/PERF_SQLSTATEMENT.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from com.main1.pythondepmanagement_1.config.ConfigStore import *
from com.main1.pythondepmanagement_1.udfs.UDFs import *

def PERF_SQLSTATEMENT(spark: SparkSession, in0: DataFrame) -> (DataFrame):
    in0.createOrReplaceTempView("in0")
    df1 = spark.sql("select * from in0 where c_int > -1000")

    return df1
